### ISSUE TITLE
Get rid of Master

### ISSUE THREAD

**Issue by dave07747** [Math-Game-MCA/Math-Game#9]
Master just straight up doesn't work and won't work. I propose that we set our current develop as master, since we know it works

**Comment by rfblue2** [comment]
The rationale for having a master branch was to house a stable version of the game.  Not sure why it would be unstable at this point, but if you deem the current develop branch as stable enough to be used as a release, then you can merge it with master.

**Comment by rfblue2** [comment]
There's also severe code maintainability issues overall in the game that need to be addressed.  Perhaps merge for now and then rework develop into a more maintainable state, then turn that into the new master branch.  Seems like feature addition will continue to stall unless the code is reworked.

**Comment by dave07747** [comment]
I agree on both points, although it is still feasible to add features to the current develop. The current master just doesn't launch in all honesty, so Noah, we need to merge
